draft-ietf-acme-acme.md

```diff
@@ -40,7 +40,7 @@ normative:
 Certificates in PKI using X.509 (PKIX) are used for a number of purposes,
 the most significant of which is the authentication of domain names.  Thus,
 certificate authorities in the Web PKI are trusted to verify that an applicant
-for a certificate legitimately represents the domain name(s) in the certificate.
+for a certificate legitimately represents the domain name(s) in a certificate.
```

Comment:
how about "in certificates"?

There can't be one (`the`) certificate with multiple certificate authorities verifying.